arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/DefaultDeploymentScenarioGenerator.java

```diff
@@ -154,6 +154,16 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
         return Collections.singletonList(description);
     }
 
+    protected String handleExceptionalCases(Archive archive, String input) {
+        if (archive.getName().endsWith(".war")) {
+            if (input.startsWith("META-INF/")) {
```

Comment:
Or maybe we put it all under WEB-INF/classes, and make our yaml-finding bits smart enough to look there.